File: 2278-find-three-consecutive-integers-that-sum-to-a-given-number/find-three-consecutive-integers-that-sum-to-a-given-number.cs
public class Solution {
    public long[] SumOfThree(long num) {
        List<long> res = new List<long>();
        if((num-3)%3==0){
            long x = (num-3)/3;
            res.Add(x);
            res.Add(x+1);
            res.Add(x+2);
        }
        return res.ToArray();
    }
}

File: 2278-find-three-consecutive-integers-that-sum-to-a-given-number/find-three-consecutive-integers-that-sum-to-a-given-number.py
class Solution:
    def sumOfThree(self, num: int) -> List[int]:
        res=[]
        if (num-3)%3==0:
            x=(num-3)//3
            res.append(x)
            res.append(x+1)
            res.append(x+2)
        return res